cadence/contracts/flix_registry.cdc

```diff
@@ -0,0 +1,172 @@
+access(all) 
+contract FLIXRegistry {
+
+    access(all)
+    event ContractInitialized()
+
+    access(all)
```

Comment:
imho all of these events need to include the name of this registry and the owner of the registry and after talking with dete in the nft standard meeting maybe even the uuid of the registry resource.

Reply:
Agreed, without the name/path and owner being included in the events I can't query the FLIX from the information currently provided in the event

Reply:
name of the registry, owner and uuid is included now in 3 events: Published, Deprecated, Removed